src/main/java/racing/view/ResultView.java

```diff
@@ -4,32 +4,61 @@
 import racing.Racing;
 
 import java.util.List;
+import java.util.stream.Collectors;
+
+import static racing.view.ViewConstant.*;
 
 /**
  * @author han
  */
 public class ResultView {
-    private String message = "실행 결과";
 
     public void print(Racing racing) {
-        System.out.println(this.message);
+        System.out.println(RESULT_MESSAGE);
 
         for (int i = 0; i < racing.getAttempts(); i++) {
             List<Car> cars = racing.getLogs().get(i);
+            printStep(cars);
+        }
+        printWinner(racing);
+    }
 
-            for (Car car : cars) {
-                String step = createSteps(car.getStep());
-                System.out.println(step);
-            }
-            System.out.println();
+    private void printStep(List<Car> cars) {
+        for (Car car : cars) {
+            String step = createResult(car);
+            System.out.println(step);
         }
+        System.out.println();
     }
 
-    private String createSteps(int count) {
+    private String createResult(Car car) {
+        int count = car.getStep();
+        String name = car.getName();
+
         StringBuilder sb = new StringBuilder();
+        sb.append(name + " : ");
```

Comment:
`+`를 이용하여 문자열을 연결하면 StringBuilder의 장점이 사라지는 것 같아요.
```suggestion
        sb.append(name);
        sb.append(" : ");
```